### ISSUE TITLE
request to add "Post-Traumatic Stress Disorders"

### ISSUE THREAD

**Issue by hpan67** [obophenotype/human-phenotype-ontology#4940]
**Preferred term label:**
Post-Traumatic Stress Disorders

**Synonyms**
PTSD

**Definition (free text, please give PubMed ID)**
NCIm Definition: An anxiety disorder precipitated by an experience of intense fear or horror while exposed to a traumatic (especially life-threatening) event. The disorder is characterized by intrusive recurring thoughts or images of the traumatic event; avoidance of anything associated with the event; a state of hyperarousal and diminished emotional responsiveness. These symptoms are present for at least one month and the disorder is usually long-term.
(NCIm Concept Unique Identifier (CUI): C0038436; NCIt Code: C3389; MeSH ID D013313

**Parent term (use hpo.jax.org/app)**
Related HPO parent term: Behavioral Abnormality, HP:0000708 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Post-Traumatic Stress Disorders

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-7166-108X

**Comment by LCCarmody** [comment]
This is currently in our disease ontology, MONDO,  http://purl.obolibrary.org/obo/MONDO_0005146
Maybe we could flesh out some of the symptoms like 'hyperarousal' and 'diminished emotional responsiveness'.  Emotional blunting HP:0030213  seems like a good synonym for the latter, maybe add as a synonym?  @pnrobinson

**Comment by pnrobinson** [comment]
I agree with @LCCarmody  that PTSD is a disease rather than a phenotypic feature. Let's make sure that the HPO has sufficent phenotypic features to represent the PTSD phenotypic spectrum. There seem to be many good reviews, but this one might be a good starting point:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4663500/

**Comment by pnrobinson** [comment]
I have added some phenotypes asssociated with PTSD, including  Intrusion symptom (and child terms), Avoidance of stimuli associated with traumatic event, and Dissociative reaction
